lrpar/src/lib/ctbuilder.rs

```diff
@@ -297,6 +300,12 @@ where
         self
     }
 
+    /// If set to true, [CTParserBuilder::build] will return an error if the grammar contains warnings.
```

Comment:
It would be good to document the default case (I think the other methods do).

Reply:
Should be fixed in [1e6c7d3](https://github.com/softdevteam/grmtools/pull/354/commits/1e6c7d3556741f4c9dbad09dce4b5bef2f28e5b8)